manticore/ethereum/manticore.py

```diff
@@ -1953,3 +1969,100 @@ def worker_finalize(q):
 
         for proc in report_workers:
             proc.join()
+
+    def set_lazy_evaluation(self, flag):
+        """
+        Set lazy evaluation flag (only EVM support)
+        :return:
+        """
+        self._lazy_evaluation = flag
+
+    def get_lazy_evaluation(self) -> bool:
```

Comment:
As before, a property would be perfect here. As for `set_lazy_evaluation`, you could convert that to a property as well, but I think this is intended to be user-facing, and most users will trust `manticore.set_lazy_evaluation(False)` more than `manticore.lazy_evaluation = False`. That said, to make it even more explicit, you could consider splitting this up into `enable_lazy_evaluation` and `disable_lazy_evaluation` methods.

Reply:
Fixed, now we have `enable/disable_lazy_evaluation`.